.circleci/config.yml

```diff
@@ -65,8 +65,11 @@ jobs:
   test-browser:
     <<: *test-defaults
     environment:
-      - TEST_TARGET=browser
-      - EMSCRIPTEN_BROWSER=0
+      # browser.test_sdl_ttf_render_text_solid depends on Arial from Microsoft
+      - TEST_TARGET=browser skip:browser.test_sdl_ttf_render_text_solid
+      - TEST_DEPENDENCY=firefox
+      - EMSCRIPTEN_BROWSER=firefox -headless
```

Comment:
this doesn't need quotes on the stuff after the `"`? honest question, I have no idea :)

Reply:
It works without quotes on config.yml as each line defines exactly one variable. Dockerfile does require quotes, though.